compatibility_server/pip_checker.py

```diff
@@ -372,7 +372,7 @@ def _check(self, container: docker.models.containers.Container):
             command,
             stdout=True,
             stderr=True,
-            raise_on_failure=False)
+            raise_on_failure=True)
```

Comment:
I don't think that this will work (see the comment 7 lines up).

Reply:
Yep and probably we could just raise an error for that case, then it will return a 500 and will not be treated as `INSTALL_ERROR`.

Reply:
And I have tested that if I set the docker timeout to a very short time like 1 second, I could reproduce this.